backend/cfg/vectorize.ml

```diff
@@ -255,8 +832,24 @@ let cfg ppf_dump cl =
   if !Flambda_backend_flags.dump_vectorize
   then Format.fprintf ppf_dump "*** Vectorization@.";
   let cfg = Cfg_with_layout.cfg cl in
-  let dependency_graph = Dependency_graph.from_cfg cfg in
-  if !Flambda_backend_flags.dump_vectorize
-  then Dependency_graph.dump ppf_dump dependency_graph cl;
+  let layout = Cfg_with_layout.layout cl in
+  DLL.iter layout ~f:(fun label ->
```

Comment:
I followed the dump of Cfg_with_layout because the basic blocks appear in that order in the cfg dumped before and after vectorize. Would it be a bit confusing if the extra information appears in a different order as the Cfgs themselves?

Reply:
ah, makes sense, thanks!